input/postgres/relation_column_stats.go

```diff
@@ -33,7 +46,7 @@ func GetColumnStats(ctx context.Context, logger *util.Logger, db *sql.DB, global
 				server.SelfTest.HintDbCollectionAspect(dbName, state.CollectionAspectColumnStats, "Please set up"+
 					" the monitoring helper function pganalyze.get_column_stats (%s)"+
 					" or connect as superuser to get column statistics for all tables.", selftest.URLPrinter.Sprint("https://pganalyze.com/docs/install/troubleshooting/column_stats_helper"))
-				logger.PrintInfo("Warning: Limited access to table column statistics detected in database %s. Please set up"+
+				logger.PrintWarning("Limited access to table column statistics detected in database %s. Please set up"+
```

Comment:
It looks like this change is causing the [test failure](https://github.com/pganalyze/collector/actions/runs/11263876700/job/31322741758). Thoughts on how to handle this? I could back out the change, but AFAIK this should really be a warning.

Reply:
The collector integration test output is a mess, but the message [is there](https://github.com/pganalyze/collector/actions/runs/11263876700/job/31322741758#step:6:7828):

> expected no warnings in collector log; test failed:

This is due to the intentional check for warnings [here](https://github.com/pganalyze/collector/blob/main/integration_test/Makefile#L288). So I think our options are:

 1. Leave this log message at Info level
 2. Update guided setup so that column stats helpers are installed, avoiding the warning
 3. Drop that check in the integration test

I think (2) is the best course of action, but that's clearly beyond the scope of this PR. I think we should do (1) for now and do (2) as a follow-up and then update the log level. For what it's worth, it looks like we have a bunch of other `PrintInfo` calls that start their output with "Warning" or "WARNING"...